lib/WIFI/WIFI.cpp

```diff
@@ -56,6 +56,7 @@
  *****************************************************************************/
 
 WIFI::WIFI() :
+    m_isStaAvailable(false),
```

Comment:
Order must be equal to the definitions in the class!

Reply:
Is the order currently incorrect?